public/app/features/scopes/tests/tree.test.ts

```diff
@@ -133,12 +136,33 @@ describe('Tree', () => {
     await openSelector();
     await expandResultCloud();
     await selectResultCloudDev();
-    expectResultCloudDevSelected();
-    expectResultCloudOpsNotSelected();
 
+    // Verify the content of the scopes selector input
+    expectScopesSelectorValue('Dev');
+
+    // Single leaf node links always apply the scope, hence we need to open the selector again
+    await openSelector();
     await selectResultCloudOps();
-    expectResultCloudDevNotSelected();
-    expectResultCloudOpsSelected();
+    expectScopesSelectorValue('Ops');
+  });
+
+  it('Can only select one selectable container at a time', async () => {
```

Comment:
We didn't have a test for selectable containers with radio buttons previously. Given that we added the link button type, we now need this to verify that a selection is unique before pressing apply.